core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -1360,6 +1363,11 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             wasm_set_exception(module, "indirect call type mismatch");
             goto got_exception;
           }
+#if WASM_ENABLE_TAIL_CALL != 0
+          opcode = *(frame_ip - 1);
```

Comment:
The frame_ip has been changed in line 1312 to 1314, *(frame_i - 1) is not opcode again. This line should be moved to the beginning of HANDLE_OP, like classic-interp.